README.md

```diff
@@ -224,7 +224,8 @@ var cassanKnex = require("cassanknex")({
   },
   exec: { // default is '{}'
     prepare: false // default is 'true'
-  }
+  },
+  awsKeyspace: false // default is 'false'
```

Comment:
👍🏼 nice work adding these docs to the README, could you also update the `<DRIVER_OPTIONS>` section above? They're under https://github.com/azuqua/cassanknex/blob/master/README.md#as-a-query-generator

Reply:
I missed the <DRIVER_OPTIONS> section, thats why i added it here.
Thank you, added to driver options aswell